### PR TITLE
Docs update, minor refactoring

### PR DESCRIPTION
**Pull request by karolh2000**
* The log analytics docs update related to the script installation.
* Minor refactoring.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ git submodule foreach git pull origin master
      * [Kubernetes](#kubernetes)
      * [User Installation](#user-installation)
      * [Root Installation](#root-installation)
+     * [Script Installation](#script-installation)
```
Comment (karolh2000) [inline README.md:28]: Not sure about the section name, “Script installation” is kind of misleading,  we're not really installing scripts but fluentd + plugins. @Turhsus @MahithaB  any ideas on how to name this section.

Comment (Turhsus) [reply README.md:28]: Why not just that? (#fluent-plugin-installation)

File: fluentd-installer/scripts/linux/log-vendors/fluentd-splunk-installer.sh [hunk 2]
```diff
@@ -62,6 +62,7 @@ shared_config_questions() {
     update_fluentd_config_file_headless "$TEMP_FOLDER/$fluentd_splunk_conf_name" "#use_ssl" "use_ssl" $fluentd_as_service
   fi
   # configure CA file
+  echo
```
Comment (Turhsus) [inline fluentd-installer/scripts/linux/log-vendors/fluentd-splunk-installer.sh:65]: Whats this Echo statement doin?

**Review by Turhsus — APPROVED** [review]
Left one comment on what looks like a stray echo, but otherwise LGTM

**Pull request merged**